### DISCUSSION TITLE
SCRIPT Question: Redirect to MAIN MENU page

### DISCUSSION THREAD

**Discussion by UBWH** [arendst/Tasmota#20362] [Q&A] [answered]
I am successfully using the Button command like this in a **SUB** webpage (>w1)

```
bu(bumqp "Done" "Save")
```

Is there a smart way to have the Script redirect the user to the Main Menu page when that button is pressed? (instead of displaying a button saying [ Done ] )

In javascript .. something like this would work
```
<script>window.onload = function() { window.location.href = "/";}</script>
}
```

**Comment by gemu2015** [comment]
i use this to return to main menu
`$<p><form onclick='javascript:history.back();return false;'   method='get' > <button>Main menu</button></form></p>`

**Comment by UBWH** [comment]
Thanks... but that seems to get back (to Main) if user clicks [ Main Menu ] button

I am trying to get browser to go back to Main when user clicks this buttom

```
bu(bumqp "Done" "Save")
```

**Reply by gemu2015** [reply, marked as answer]
since bu() is a macro, you may simply solve the macro and insert the history.back there:
```
<div>
    <button type='submit' style="width:100%%" onclick='seva(0,"bumqp"); history.back()' >Save</button>
</div>
```

**Comment by UBWH** [comment]
Thanks. Worked well.
